docs/how-to/required-software.md

```diff
@@ -0,0 +1,96 @@
+# How To - Required software
+
+## Introduction
+
+This guide shows you how to setup a Mac for iOS development.
+
+You will need a Mac in order to run this. It will not work on a Windows or Linux device.
+
+## Xcode
+
+You'll need an Apple ID in order to download Xcode. Download it using [this link](https://developer.apple.com/download/applications). You will need to become an Apple Developer (you will be prompted to do this, and agree to their terms and conditions) - this is free.
```

Comment:
Open question but is there any way to make this more prominent in markdown than this?

Reply:
(the link to Xcode download, that is)

Reply:
Do you mean like make it bold or something?

Reply:
Yeah just wondering if there's anything to do to make that link more noticeable, not sure if there is 🙂

Reply:
I've made it bold, but I'm not sure it looks any better or clearer. We could look at highlighting or using italics instead?

Reply:
Looks better now, don't think it's a huge point so happy to proceed as